bigchaindb/pipelines/block.py

```diff
@@ -1,6 +1,6 @@
 """This module takes care of all the logic related to block creation.
```

Comment:
I renamed this class to now be called `BlockPipeline`, which is hopefully alright (@vrde and @rhsimplex).
Shall we rename the file as well?